CONTRIBUTING.md

```diff
@@ -74,37 +75,34 @@ export class AppModule {
 }
 ```
 - if you are using `ng-bootstrap` as peer dependency, you have to add `import '@angular/localize/init'` in your `app/polyfill.ts`
-- build your application in prod mod with `ng build --prod --output-hashing=none search-wc` at the root of the project. 
 
-To use your webcomponent in a website, you can
-- merge all the build output javascript files into one and GZIP it
-```shell script
-cat dist/search-wc/{runtime-es2015,polyfills-es2015,main-es2015}.js | gzip > gn-search-snapshot.js.gz
-```
-- include it in your HTML file
-```angular2html
-...
-    <script src="gn-search-snapshot.js"></script>
-...
-    <gn-search-snapshot></gn-search-snapshot>
-```
-and run your html file in a web server
+#### Test your Web Component
```

Comment:
I think the documentation is much better now, but it would be better suited in `webcomponents/README` instead of `CONTRIBUTING`. Especially the general usage part.